plugins/catalog-backend-module-bitbucket-cloud/src/service/BitbucketCloudCatalogModule.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  configServiceRef,
+  createBackendModule,
+  loggerServiceRef,
+  loggerToWinstonLogger,
+  schedulerServiceRef,
+} from '@backstage/backend-plugin-api';
+import { catalogProcessingExtensionPoint } from '@backstage/plugin-catalog-node';
+import { BitbucketCloudEntityProvider } from '../BitbucketCloudEntityProvider';
+
+/**
+ * Bitbucket Cloud module for the Catalog plugin
+ *
+ * @alpha
+ */
+export const bitbucketCloudCatalogModule = createBackendModule({
+  moduleId: 'bitbucketCloud',
```

Comment:
btw: is there a certain convention for the naming already?

plugins/catalog-backend-module-github/src/module.ts
```ts
export const githubEntityProviderCatalogModule = createBackendModule({
  pluginId: 'catalog',
  moduleId: 'github-entity-provider',
```
here, I would have probably expected to see just `github` as in the module name. Also, it uses hyphen separators instead of camel-case like I did here. Should I adjust it?

plugins/scaffolder-backend/src/extension/ScaffolderCatalogModule.ts
```ts
export const scaffolderCatalogModule = createBackendModule({
  moduleId: 'scaffolder.module',
  pluginId: 'catalog',
```
here, I would not have expected the suffix `.module`

Reply:
Hi @pjungermann!

Good catch with the naming. We need to add some guidance about this in the documentation. The `scaffolderCatalogModule`should not have module in the ID, you're right 😅 
We have used dots as seperators previously so perhaps go with dots instead. Then should it be camel cased? Like `github.entityProvider`? @Rugvip @freben WDYT?

Reply:
Suggesting ^ in #13972

Reply:
makes sense to me 👍 

At the moment, the plugin only exports the provider and provides nothing else, but I can still add the suffix to the moduleId, etc

`bitbucketCloudEntityProviderCatalogModule` 

Would you prefer this?

Reply:
Let's make it

```ts
export const bitbucketCloudEntityProviderCatalogModule = createBackendModule({
  pluginId: 'catalog',
  moduleId: 'bitbucketCloudEntityProvider',
 ```

Reply:
adjusted